README.md

```diff
@@ -1,2 +1,15 @@
 # tortoise-lang
 The Tortoise Language (inspired by an exercise in The Pragmatic Programmer)
+
+Running code analysis tools:
+We have three code analysis tool in tools.bash file, pytest, black, and pylint.
+Every time we change someting in the code these analysis tools should be run.
+To run the scripts:
```

Comment:
Do you want lines 5, 6, and 7 to be one paragraph, or separate? Because this is currently ambiguous, different renderers will display it differently.

Reply:
You still need to address this comment.